pkg/drain/cordon.go

```diff
@@ -62,10 +62,7 @@ func NewCordonHelperFromRuntimeObject(nodeObject runtime.Object, scheme *runtime
 // or false when no change is needed
 func (c *CordonHelper) UpdateIfRequired(desired bool) bool {
 	c.desired = desired
-	if c.node.Spec.Unschedulable == c.desired {
-		return false
-	}
-	return true
+	return !(c.node.Spec.Unschedulable == c.desired)
```

Comment:
```suggestion
	return c.node.Spec.Unschedulable != c.desired
```

Reply:
Thanks for your suggestion. I changed it.
Sorry about this mistake.